### ISSUE TITLE
Error when resolved promise returns null

### ISSUE THREAD

**Issue by steida** [pburtchaell/redux-promise-middleware#20]
This line expects existing `resolved`, and trying to read its meta.
https://github.com/pburtchaell/redux-promise-middleware/blob/master/src/index.js#L34

Which makes this error:
```
Unhandled rejection TypeError: Cannot read property 'meta' of undefined
    at eval (eval at <anonymous> (http://localhost:8080/build/app.js:2779:2), <anonymous>:52:24)
    at eval (eval at <anonymous> (http://localhost:8080/build/app.js:1591:2), <anonymous>:21:50)
    at dispatch (eval at <anonymous> (http://localhost:8080/build/app.js:2797:2), <anonymous>:46:18)
```

My code

```
export function login() {
  return ({firebase}) => ({
    type: 'LOGIN',
    payload: {
      promise: firebase.authWithOAuth('facebook', getOptions('facebook'))
        .then(auth => {
          const user = createUserFromFacebook(auth);
          return firebase.set(['users', user.id], user.toJS());
        })
    }
  });
}
```

Btw, I don't understand why resolved is supposed to by always defined.

**Comment by tomatau** [comment]
I'm in the habit of always resolving something so that my state can be updated with information but I understand that the action.type is enough to achieve this. Have made a PR which should fix the issue

**Comment by pburtchaell** [comment]
Alright, this should be fixed in `redux-promise-middleware@2.2.0`. Let us know if that works for you @steida.

**Comment by steida** [comment]
Yes, it works. Thank you.
